recipes/oatpp-libressl/all/conanfile.py

```diff
@@ -0,0 +1,90 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class OatpplibresslConan(ConanFile):
+    name = "oatpp-libressl"
+    license = "Apache-2.0"
+    homepage = "https://github.com/oatpp/oatpp-libressl"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "oat++ libressl library"
+    topics = ("conan", "oat++", "oatpp", "libressl")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake", "cmake_find_package"
```

Comment:
upstream uses pkg-config for libressl 💭 is it required?

https://github.com/oatpp/oatpp-libressl/blob/cd2e9a515131e5e7dc043c591e952e12cd63db2c/CMakeLists.txt#L98

Reply:
I think yes, not everybody will use conan to build oatpp-libressl and without it there are problems with building.

Reply:
But does C3i _need_ it? From the logs https://github.com/conan-io/conan-center-index/pull/3420#issuecomment-722944075 it seems to work just fine